testing/tests/DevExpress.exporter/jspdfParts/jspdf.dataGrid.tests.js

```diff
@@ -0,0 +1,79 @@
+import $ from 'jquery';
+import { jsPDF } from 'jspdf';
+import 'jspdf-autotable';
+
+import { isFunction } from 'core/utils/type';
+import { JSPdfDataGridTestHelper } from './jspdfTestHelper.js';
+import { exportDataGrid } from 'pdf_exporter';
+
+import 'ui/data_grid/ui.data_grid';
+
+import 'common.css!';
+import 'generic_light.css!';
+
+QUnit.testStart(() => {
+    const markup = '<div id=\'dataGrid\'></div>';
```

Comment:
I suggest to use double quotes instead of \' sequence, it's much easier to read it:

const markup = '<div id="dataGrid"></div>';